.github/workflows/validate-nginx-config.yml

```diff
@@ -0,0 +1,74 @@
+name: Validate Nginx Configuration
+on:
+  workflow_call:
+    inputs:
+      templatedir: # Ref: https://github.com/nginxinc/docker-nginx-unprivileged/blob/main/entrypoint/20-envsubst-on-templates.sh#L32
```

Comment:
Why you prefer `templatedir` instead of `template_dir`?

Reply:
@joshuagibeonm updated to use snake_case as our convention. Following pre-existed action and workflow